pandas/core/arrays/integer.py

```diff
@@ -573,6 +564,8 @@ def _maybe_mask_result(self, result, mask, other, op_name):
 
     @classmethod
     def _create_arithmetic_method(cls, op):
+
+        # @ops.unpack_and_defer
```

Comment:
@jreback using the decorator here breaks a bunch of tests, specifically ones operating with DataFrame.  Why does this return NotImplemented for Series/Index but not DataFrame?

Reply:
Below, why use `other = other.item()` instead of `item_from_zerodim`?